manifests/ssltools/nssdb.pp

```diff
@@ -0,0 +1,35 @@
+# Creates a nssdb
+class certs::ssltools::nssdb inherits certs {
```

Comment:
Could you implement it as a non-inheriting class with parameters? That would make testing easier and faster.